guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -16,24 +16,38 @@
     "<ul class=\"toc-item\">\n",
```

Comment:
<ul><li><a href="https://nbviewer.jupyter.org/github/Esri/arcgis-python-api/blob/6b709f913f3b74d2a27cf0455792d75a53a1a6eb/guide/14-deep-learning/how-named-entity-recognition-works.ipynb#Introduction" target="_blank">Introduction</a></li><li class="ql-indent-1"><a href="https://nbviewer.jupyter.org/github/Esri/arcgis-python-api/blob/6b709f913f3b74d2a27cf0455792d75a53a1a6eb/guide/14-deep-learning/how-named-entity-recognition-works.ipynb#What-is-unstructured-text?" target="_blank">What is unstructured text?</a></li><li class="ql-indent-1"><a href="https://nbviewer.jupyter.org/github/Esri/arcgis-python-api/blob/6b709f913f3b74d2a27cf0455792d75a53a1a6eb/guide/14-deep-learning/how-named-entity-recognition-works.ipynb#What-is-Named-Entity-Recognition?" target="_blank">What is Named Entity Recognition?</a></li><li><a href="https://nbviewer.jupyter.org/github/Esri/arcgis-python-api/blob/6b709f913f3b74d2a27cf0455792d75a53a1a6eb/guide/14-deep-learning/how-named-entity-recognition-works.ipynb#Prerequisites" target="_blank">Prerequisites</a></li><li class="ql-indent-1"><a href="https://nbviewer.jupyter.org/github/Esri/arcgis-python-api/blob/6b709f913f3b74d2a27cf0455792d75a53a1a6eb/guide/14-deep-learning/how-named-entity-recognition-works.ipynb#Entity-recognition-with-spaCy" target="_blank">Entity recognition with spaCy</a></li><li class="ql-indent-1"><a href="https://nbviewer.jupyter.org/github/Esri/arcgis-python-api/blob/6b709f913f3b74d2a27cf0455792d75a53a1a6eb/guide/14-deep-learning/how-named-entity-recognition-works.ipynb#Entity-recognition-with-transformer" target="_blank">Entity recognition with transformer</a></li></ul><p class="ql-indent-1"><br></p><p> </p><p>The prerequisites section should not have the following sub sections?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>